tests/t196_chrome_taskname.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+from runtest import TestBase
+import subprocess as sp
+
+TDIR='xxx'
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'taskname', ldflags='-pthread', result="""
+{"traceEvents":[
+{"ts":0,"ph":"M","pid":4694,"name":"process_name","args":{"name":"t-taskname"}},
+{"ts":13453314717.085,"ph":"B","pid":4694,"name":"main"},
+{"ts":13453314717.245,"ph":"B","pid":4694,"name":"task_name1"},
+{"ts":13453314717.814,"ph":"B","pid":4694,"name":"prctl"},
+{"ts":0,"ph":"M","pid":4694,"name":"process_name","args":{"name":"foo"}},
```

Comment:
Is it ok for chrome to have 0 "ts" here?

Reply:
Yes, I got it from the normally recorded data of existing chrome tracing infra, so I think it's correct.